ansible-playbooks/roles/deploy-acm-stack/tasks/main.yml

```diff
@@ -147,6 +147,27 @@
     - "cert-manager-config"
     - "openshift-config"
 
+- name: Wait for ACM InstallPlan creation
+  kubernetes.core.k8s_info:
+    kind: InstallPlan
+    api_version: v1alpha1
+    namespace: open-cluster-management
+  register: acm_installPlan  
+  until:
+    - acm_installPlan.resources.0 is defined
+  retries: 15
+  delay: 20
+
+- name: Approve the ACM InstallPlan
+  kubernetes.core.k8s:
+    api_version: v1alpha1
+    kind: InstallPlan
+    namespace: open-cluster-management
+    name: "{{ acm_installPlan.resources.0.metadata.name }}"
```

Comment:
in case of multiple install plans, this one is the latest/more recent? or always the first?

Reply:
For fresh install, we add the `startingCSV` to pin the install version to v2.5.0. In case v2.5.1 released and available, another install plan will created for 2.5.1. But `acm_installPlan.resources.0` will only point to first install plan that is we defined in Subscription.

Reply:
I think it would make sense to approve the last one, it means that in case of upgrade we could:
1) do the PR
2) run the playbook to approve it
What do you think?

Reply:
Then approve the last one will make the `startingCSV ` useless here. And I remember in our MVP goals for ACM, it mentioned:
- !> 1 z release behind ACM product release
- !> 1 minor release behind ACM product release

So it means we should stay 1 version behind the y and z release. 

For upgrade as you mentioned:
1. Do the PR
2. Approve the install plan:
    - Approve it in console UI 
    - Run another playbook created specific to do the acm upgrade with var target the install plan we want approve.

Any comments is appreciated.